runtime/src/main/java/org/corfudb/runtime/object/transactions/OptimisticTransactionalContext.java

```diff
@@ -56,6 +49,13 @@
     private final Set<ICorfuSMRProxyInternal> modifiedProxies =
             new HashSet<>();
 
+    /** Keep the version (at global snapshot time)
```

Comment:
I see, I got version and version number confused. I probably should have read through the code more carefully. Unfortunately, I'm not sure it improves performance at all... not really sure why.